### PR TITLE
Patch out examples w/ missing dependencies

### LINKED ISSUE #222: Dev Screens not showing/working
Even though opting in for the Ignite Development Screens they are not included. After adding them with "ignite add dev-screens" it fails with the following error:

error: bundling failed: Error: Unable to resolve module `../../ignite/DevScreens/DevscreensButton.js` from `C:\InnoTact\AndrossType\App\Containers\LaunchScreen.js`: The module `../../ignite/DevScreens/DevscreensButton.js` could not be found from `C:\InnoTact\AndrossType\App\Containers\LaunchScreen.js`. Indeed, none of these files exist:

  * `C:\InnoTact\AndrossType\ignite\DevScreens\DevscreensButton.js(.native||.android.js|.native.js|.js|.android.json|.native.json|.json)`
  * `C:\InnoTact\AndrossType\ignite\DevScreens\DevscreensButton.js\index(.native||.android.js|.native.js|.js|.android.json|.native.json|.json)`
    at ModuleResolver.resolveDependency (C:\InnoTact\AndrossType\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:161:851)
    at ResolutionRequest.resolveDependency (C:\InnoTact\AndrossType\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:91:16)
    at DependencyGraph.resolveDependency (C:\InnoTact\AndrossType\node_modules\metro\src\node-haste\DependencyGraph.js:272:4579)
    at dependencies.map.relativePath (C:\InnoTact\AndrossType\node_modules\metro\src\DeltaBundler\traverseDependencies.js:376:19)
    at Array.map (<anonymous>)
    at resolveDependencies (C:\InnoTact\AndrossType\node_modules\metro\src\DeltaBundler\traverseDependencies.js:374:16)
    at C:\InnoTact\AndrossType\node_modules\metro\src\DeltaBundler\traverseDependencies.js:212:33
    at Generator.next (<anonymous>)
    at step (C:\InnoTact\AndrossType\node_modules\metro\src\DeltaBundler\traverseDependencies.js:297:313)
    at C:\InnoTact\AndrossType\node_modules\metro\src\DeltaBundler\traverseDependencies.js:297:473
 BUNDLE  [android, dev] ./index.js ░░░░░░░░░░░░░░░░ 4.0% (117/651), failed.

My current versions are:
System
  platform           win32
  arch               x64
  cpu                4 cores                        Intel(R) Core(TM) i7-6560U CPU @ 2.20GHz
  directory          C:\InnoTact\AndrossTypeScipt

JavaScript
  node               10.5.0       C:\Program Files\nodejs\node.EXE
  npm                6.1.0        C:\Program Files\nodejs\npm.CMD
  yarn               1.3.2        C:\Users\Joel Rudsberg\AppData\Roaming\npm\yarn.CMD

React Native
  react-native-cli   2.0.1
  app rn version     0.55.1

Ignite
  ignite             2.1.2        C:\Users\Joel Rudsberg\AppData\Roaming\npm\ignite.CMD

Android
  java               -            C:\Program Files\Java\jdk1.8.0_172\bin\java.EXE
  android home       -            C:\Users\Joel Rudsberg\AppData\Local\Android\Sdk

### PR DESCRIPTION
**Pull request by morgandonze**
Also necessary to fix https://github.com/infinitered/ignite-andross/issues/222:
https://github.com/infinitered/ignite-andross/pull/238

With this PR, `ignite-dev-screens` will either remove or restore the lines which import examples (either `ignite-animatable` or `ignite-vector-icons`), depending on whether the necessary modules are present.

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -46,6 +46,44 @@ const add = async function (context) {
...
   }
 
+  // Patch for animatable
+  const pluginExamplesScreen = `${process.cwd()}/ignite/DevScreens/PluginExamplesScreen.js`
+  try {
+    const animatableExample = `${process.cwd()}/node_modules/ignite-animatable`
+    if (!filesystem.exists(animatableExample)) {
+      ignite.patchInFile(pluginExamplesScreen, {
+        replace: 'import \'../Examples/Components/animatableExample.js\'',
+        insert: '// animatableExample removed - ignite-animatable not installed [DO NOT REMOVE THIS LINE!]'
```
Comment (jamonholmgren) [inline plugin.js]: We probably don't need the `DO NOT REMOVE THIS LINE` part, right?

Comment (morgandonze) [reply plugin.js]: We don't, but if they remove it, then install `ignite-animatable`, then if they run `ignite add ignite-dev-screens`, the `import` line won't be restored. (Though I realize I would need to make the line in the `else` block match this.) I'll go ahead and remove it though.

Comment (jamonholmgren) [reply plugin.js]: We can probably fix that without the comment. Just have it import below something else.

**Pull request merged**